library/streams/src/main/java/it/geosolutions/io/output/adapter/OutputStreamAdapter.java

```diff
@@ -65,4 +65,8 @@ public void write(byte[] b, int off, int len) throws IOException {
     public void write(int b) throws IOException {
         stream.write(b);
     }
+
+    public ImageOutputStream getWrappedStream() {

```

Comment:
Javadoc? :-D

Reply:
here it is (I have also added it for the ImageInputStream one)

Reply:
Thanks!